plotly/plotly/plotly.py

```diff
@@ -1209,14 +1275,20 @@ def parse_grid_id_args(cls, grid, grid_url):
 
     @classmethod
     def response_handler(cls, response):
-
         try:
             response.raise_for_status()
         except requests.exceptions.HTTPError as requests_exception:
-            plotly_exception = exceptions.PlotlyRequestError(
-                requests_exception
-            )
-            raise(plotly_exception)
+            if (response.status_code == 404 and
+                    get_config()['plotly_api_domain']
+                    != 'https://api.plot.ly'):
+                raise exceptions.PlotlyError(
+                    " "
+                )
+            else:
+                plotly_exception = exceptions.PlotlyRequestError(
+                    requests_exception
+                )
+                raise(plotly_exception)
```

Comment:
otherwise, I think we should just re-raise `requests_exception`. thoughts @theengineear ?